xml/operations-maintenance-swift-add_swift_pac_node.xml

```diff
@@ -67,17 +67,17 @@ git checkout site</screen>
      In the above example, only one new server
      <emphasis role="bold">swpac6</emphasis> is added. However, you can add
      multiple servers by providing the server details in the
-     <literal>servers.yml</literal> file.
+     <filename>servers.yml</filename> file.
     </para>
     <para>
      In the entry-scale configurations there is no dedicated Swift PAC cluster.
      Instead, there is a cluster using servers that have a role of
-     <literal>CONTROLLER-ROLE</literal>. You cannot add
-     <literal>swpac4</literal> to this cluster because that would change the
-     <literal>member-count</literal>. If your system does not already have a
-     dedicated Swift PAC cluster you will need to add it to the configuration
-     files. For details on how to do this, see
-     <xref linkend="topic_rvj_21c_jt"/>.
+     <literal>CONTROLLER-ROLE</literal>. You cannot add additional nodes
+     dedicated to Swift PAC, becuase that would change the
+     <literal>member-count</literal> of the entire cluster. In that case, if
+     your system does not already have a dedicated Swift PAC cluster you will
+     need to add it to the configuration files. For details on how to do this,
```

Comment:
Rewording suggestion: "...Dedicate Swift Pac cluster, you need to add it to the configuration files."